terraform/bind/main.tf

```diff
@@ -27,9 +27,9 @@ resource "aws_iam_user_policy" "user_policy" {
         "ses:SendEmail",
         "ses:SendRawEmail"
       ],
-      "Resource": "${var.domain_arn}"
+      "Resource": "*"
```

Comment:
I don't think this is correct... See https://github.com/GSA/data.gov/issues/3810#issuecomment-1117997318

Reply:
It's reverted in #16